CMake/FindPugixml.cmake

```diff
@@ -0,0 +1,14 @@
+find_path(pugixml_INCLUDE_DIRS pugixml.hpp)
+find_library(pugixml_LIBRARIES NAMES pugixml)
+mark_as_advanced(pugixml_INCLUDE_DIRS pugixml_LIBRARIES)
+
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(pugixml REQUIRED_VARS pugixml_LIBRARIES pugixml_INCLUDE_DIRS)
+
+if(Pugixml_FOUND AND NOT TARGET pugixml)
+  add_library(pugixml UNKNOWN IMPORTED)
+  set_target_properties(pugixml PROPERTIES
+    IMPORTED_LOCATION "${pugixml_LIBRAIRIES}"
```

Comment:
`LIBRAIRIES`?

Reply:
Whoops, stupid typo. Thanks for catching this.